scripts/plot.py

```diff
@@ -15,38 +15,34 @@ def plot_latency(fname, fig=None, ax1=None, ax2=None):
     tx_vals = []
     fresh = False  # is this the plot of all data, or just a single plot
 
-    if (fig == None):  # create new axes if necessary
+    if not fig: # create new axes if necessary
         fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(12, 5))
         fig.suptitle(fname+" performance data")
         fresh = True
     local_file = fname.split('/')
     local_file = local_file[-1].split('_')
-    id = local_file[-1].split('.')[0]
-    nm = 'loadgen_' + id
+    iden = local_file[-1].split('.')[0]
```

Comment:
Some of these name changes seem quite arbitrary -- I'm curious does the linter prevent us from using names like `id`, `i`, `fin`

Reply:
```console
scripts/plot.py:24:4: W0622: Redefining built-in 'id' (redefined-builtin)
scripts/plot.py:80:4: W0621: Redefining name 'fin' from outer scope (line 93) (redefined-outer-name)```
These were the lint warnings I addressed for these